content/video-production/live-events.mdx

```diff
@@ -43,46 +43,25 @@ Using the latest technologies and with coaching from our media team SSW can crea
 
 Live streaming can be a great way to interact with a remote audience, in addition to the live audience. It might seem as easy as operating a Facebook account at first glance but when it comes down to it, it’s still a fairly new technology and it’s still very technical, especially if you want multiple cameras and/or computer screen feeds etc. so you need an expert that knows the ins and outs and can create a steady, professional live streaming setup. That’s us!
 
-### Scott Hanselman Presentation
+#### Scott Hanselman Presentation
 
 <VideoEmbed url="https://player.vimeo.com/video/121205693" />
 
-### Live Stream Your Event
+Come
+to SSW - Hire a conference room with livestream
+-----------------------------------------------
 
-If you are already running a conference or event that you would like make available to a wider audience, we can come onsite and livestream your event. This would be suitable for conference sessions, user groups, AGMs, important company meetings, School or church events, or educational presentations.
+SSW offers a spacious and quiet training rooms in Sydney, Melboure, Brisbane, and Newcastle. 
 
-An indicative price for livestreaming an event like this would be approximately $3,000 AUD + GST per day.
+We, at SSW, utilize this hall for many of our corporate training events and we are now making it available to other businesses looking for an uncomplicated venue to hold their small events without that hassle of negotiating with large conference hall coordinators. Learn more at [SSW Chapel website](https://www.ssw.com.au/ssw/Events/Venues/Hire-Conference-Room.aspx).
 
-## Install Livestream System
 
-### Description
+An indicative price for livestreaming an event like this would be approximately $9,000 AUD + GST per day, including preproduction and  livestreaming with a dedicated Event Master and a crew of 2 SSW TV team members.
 
-SSW is one of the first software consultancies in Australia to regularly Live Stream high quality presentations. We have recognised that other businesses have a need to live stream. If you are looking at live streaming your events, then our technicians can install a system for you and train your staff to use it.
+Please call for a quote to meet your specific requirements.
```

Comment:
2. change to
Please call for a quote that meets your prerequisites.

Reply:
Done